flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/LocalExecutorITCase.java

```diff
@@ -168,16 +179,19 @@ public void testViews() throws Exception {
 				"TestView2");
 		assertEquals(expectedTables, actualTables);
 
+		// Although AdditionalView2 needs AdditionalView1, dropping AdditionalView1 first does not
+		// throw.
```

Comment:
Incomplete sentence ?

Reply:
Not really, `does not throw` means `does not throw exception`.